backend/pkg/log/server/log_service_server_impl.go

```diff
@@ -93,3 +111,9 @@ func getSeverity(severityNumber v1.SeverityNumber) model.Level {
 		return model.InfoLevel
 	}
 }
+
+func generateLogId(timeStamp time.Time, message string) string {
```

Comment:
Added a unique ID because otherwise we kept inserting the same damn logs over and over